dashboards-observability/public/components/event_analytics/explorer/explorer.tsx

```diff
@@ -410,8 +410,7 @@ export const Explorer = ({
 
       // to fetch patterns data on current query
       if (!finalQuery.match(PPL_PATTERNS_REGEX)) {
-        const patternErrorHandler = getErrorHandler('Error fetching patterns');
-        getPatterns(patternErrorHandler);
+        getPatterns(minInterval, getErrorHandler('Error fetching patterns'));
```

Comment:
I think this is reversed I believe function signature is: 
`getPatterns = (errorHandler: (error: any) => void, query?: string)`

Reply:
https://github.com/opensearch-project/observability/blob/2.4/dashboards-observability/public/components/event_analytics/hooks/use_fetch_patterns.ts#L53
Correct me if wrong

Reply:
i changed function signature, check my branch instead of 2.4
https://github.com/opensearch-project/observability/blob/bf35b586ad43cc0f2c53d9f61eaf7c1f82a0202f/dashboards-observability/public/components/event_analytics/hooks/use_fetch_patterns.ts#L75